docs/tour/hello-world.mdx

```diff
@@ -114,6 +114,12 @@ On **macOS or Linux**, you can use the official install script to download `rust
 curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
 ```
 
+Once you've installed Rust on macOS or Linux, use `rustup` to add the `wasm32-wasip1` target:
```

Comment:
```suggestion
Once you've installed the Rust toolchain, use `rustup` to add the `wasm32-wasip1` target:
```

Generally `rustup` works on windows the same way right *once it's installed*?

Might also be worth including a link to the [windows install docs](https://forge.rust-lang.org/infra/other-installation-methods.html) earlier on (i.e. *before* Rust is installed)... 

Maybe something like:

```md
On Windows, the Rust project providers [standalone installers](https://forge.rust-lang.org/infra/other-installation-methods.html) as an alternate installation method
```